src/client/components/InviteUserForm/InviteUserForm.tsx

```diff
@@ -46,28 +46,28 @@ const InviteUserForm: React.FC = () => {
       <div className="edit-user__form-item">
         <div className="edit-user__form-label">{t('common.name')}*</div>
         <input
-          className="edit-user__form-field edit-user__form-input-text-field text-input__input-field"
-          onFocus={() => setErrors({ ...errors, name: null })}
+          className="edit-user__form-field edit-user__form-input-text-field input-text"
```

Comment:
@minotogna 
> Can you please rollback user forms and anything related to users ? 

The only change was the classname: `text-input__input-field` was deleted so I replaced it with the existing `input-text`. The other changes shown in this commit were done by the pre-commit hooks.
It's rolledback now 👍